src/blocks/DetailsBlock/DetailsBlock.astro

```diff
@@ -0,0 +1,45 @@
+---
+import type { DetailsBlockFragment } from '@lib/types/datocms';
+
+interface Props {
+  block: DetailsBlockFragment
+}
+
+const { block } = Astro.props;
+const splitChar = '|';
```

Comment:
avoid abbreviations. Rather `splitCharacter` or `separator` (as that's how the argument in `string.split` is called)

Reply:
Though `char` was pretty clear, because of functions like `charAt()`, but yeah I can name it `splitCharacter`. I thought that would be more clear since `separator` is a more broader term.